doc/user_guide/psyclone_command.rst

```diff
@@ -1,9 +1,9 @@
-.. _psyclone_script:
+.. _psyclone_command:
 
-PSyclone Script
-===============
+The psyclone command
+====================
 
-The simplest way to run PSyclone is to use the ``psyclone`` script. If
+The simplest way to run PSyclone is to use the ``psyclone`` command. If
 you installed PSyclone using ``pip`` then this script should be available
```

Comment:
The word `script` and phrase `psyclone script` continues to be used a lot in this section. To be consistent with your change it would probably be better to change `script` to `command` throughout this section.

Reply:
Done